### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by unaib** [atom/atom#15851]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.21.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: Cannot read property '0' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/partial-keyup-matcher.js:57

TypeError: Cannot read property '0' of undefined
    at PartialKeyupMatcher.getMatches (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/partial-keyup-matcher.js:42:105)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:375:91)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -3:03.6.0 editor:newline (input.hidden-input)
     -3:00.7.0 intentions:highlight (input.hidden-input)
     -2:55.6.0 core:paste (input.hidden-input)
 17x -2:54.2.0 core:move-up (input.hidden-input)
     -2:52.8.0 core:move-down (input.hidden-input)
     -2:30.3.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
     -2:29.9.0 ex-mode:open (input.hidden-input)
     -2:29.5.0 core:confirm (input.hidden-input)
  5x -2:29.5.0 blur (input.hidden-input)
     -0:27.4.0 proton:toggle (input.hidden-input)
  2x -0:27.2.0 proton:chain (input.hidden-input)
     -0:26.9.0 atomatigit:toggle (atom-text-editor.editor.normal-mode.is-focused.vim-mode-plus)
 11x -0:24.1.0 blur (input.hidden-input)
  2x -0:17.8.0 vim-mode-plus:move-down (input.hidden-input)
  4x -0:16.8.0 vim-mode-plus:move-up (input.hidden-input)
  8x -0:13.8.0 blur (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.7 
atom-beautify 0.30.5 
atom-dark-fusion-syntax 2.2.0 
atom-material-syntax 1.0.6 
atom-material-ui 2.0.4 
atomatigit 1.5.5 
autocomplete-paths 2.12.0 
autoupdate-packages 1.3.1 
blame 0.12.0 
busy 0.7.0 
busy-signal 1.4.3 
easy-motion-redux 1.1.1 
ex-mode 0.18.0 
expand-region 0.3.2 
file-icons 2.1.12 
git-history 3.3.0 
git-plus 7.9.3 
golden-ratio 0.3.0 
highlight-line 0.12.0 
highlight-selected 0.13.1 
intentions 1.1.5 
language-diff 0.7.0 
lines 0.13.1 
linter 2.2.0 
linter-ui-default 1.6.10 
maximize-panes 0.2.0 
merge-conflicts 1.4.5 
move-panes 0.2.0 
nucleus-dark-ui 0.12.3 
project-plus 1.0.0 
proton-mode 0.16.0 
recent-files-fuzzy-finder 0.3.2 
relative-numbers 0.8.0 
release-notes 0.53.0 
tab-switcher 1.5.6 
theme-switch 0.5.0 
vim-mode-plus 1.9.0 
zentabs 0.8.8 
```

**Comment by rsese** [comment]
Thanks for the report!  Can you confirm a few things for us?

* what were you doing at the time of the error?
* is the error is reproducible?
* if reproducible, do you see the error in safe mode (atom --safe)?

**Comment by Ben3eeE** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/dvcrn/proton/issues/286. And it has been fixed in Atom 1.22 currently in beta /cc: https://github.com/atom/atom-keymap/pull/221

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
